components/toast/index.css

```diff
@@ -10,18 +10,60 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+/* toast/index.css
+ * 
+ * .spectrum-Toast::after is the Focus ring
+ *
+ * Component naming template:
+ * `--system-component-style-description-modifier`
+ *
+ * Example:
+ * `--spectrum-toast-spacing-text-to-button-horizontal`
+ */
+
 .spectrum-Toast {
-  --spectrum-toast-icon-padding-y: var(--spectrum-global-dimension-size-85);
-  --spectrum-toast-neutral-content-padding-top: var(
-    --spectrum-global-dimension-size-65
-  );
-  --spectrum-toast-content-padding-bottom: var(
-    --spectrum-global-dimension-size-65
-  );
-
-  --spectrum-toast-button-margin-right: var(
-    --spectrum-global-dimension-size-130
-  );
+  /* Hardcoded variables */
+  --spectrum-toast-font-weight: 400;
+
+  /* Size */
+
+  --spectrum-toast-font-size: var(--spectrum-font-size-100);
+  --spectrum-toast-corner-radius: var(--spectrum-corner-radius-100);
+
+  --spectrum-toast-line-height: var(--spectrum-line-height-100);
+  --spectrum-toast-line-height-CJK: var(--spectrum-CJK-line-height-100);
+
+  /* Space */
+
+  --spectrum-toast-spacing-icon-to-text: var(--spectrum-text-to-visual-100);
+
+  --spectrum-toast-spacing-start-edge-to-text-and-icon: var(--spectrum-spacing-300);
+  --spectrum-toast-spacing-text-and-action-button-to-divider: var(--spectrum-spacing-300);
+
+  --spectrum-toast-spacing-top-edge-to-divider: var(--spectrum-spacing-100);
+  --spectrum-toast-spacing-bottom-edge-to-divider: var(--spectrum-spacing-100);
+
+  --spectrum-toast-spacing-top-edge-to-icon: var(--spectrum-toast-top-to-workflow-icon);
+
+  --spectrum-toast-spacing-text-to-action-button-horizontal: var(--spectrum-spacing-300);
+
+  --spectrum-toast-spacing-close-button: var(--spectrum-spacing-300);
+  --spectrum-toast-spacing-edge-to-text-to-action-button-vertical: var(--spectrum-spacing-100);
+  --spectrum-toast-spacing-edge-to-text-to-action-button-vertical: var(--spectrum-spacing-100);
+
+  /* Color */
+
+  --spectrum-toast-negative-background-color-default: var(--spectrum-negative-background-color-default);
+  --spectrum-toast-positive-background-color-default: var(--spectrum-positive-background-color-default);
+  --spectrum-toast-informative-background-color-default: var(--spectrum-informative-background-color-default);
+
+  --spectrum-toast-text-and-icon-color: var(--spectrum-white);
+}
+
+@media (forced-colors: active) {
+  .spectrum-Toast {
+    border: 1px solid transparent;
```

Comment:
Maybe we could soft-code this `1px` value? A bit of a nit, not a bit priority or blocker

Reply:
Can we use the custom property defined [here](https://github.com/adobe/spectrum-css/pull/1483/files#diff-dd286afa975e8faf641af07455647fd4d891bcc2997ac531249fa23a6dad2b07R65)?

```css
border: var(--mod-toast-border-width, var(--spectrum-toast-border-width)) solid transparent;
```

Reply:
Yes. I think this is outdated. I refactored this to use that property.